config/checkstyle_sevntu_checks.xml

```diff
@@ -161,8 +161,8 @@
                       ### Field(static .*)
                       ### Field(private static final)
                       ### Field(private static .*)
-                      ### DeclareAnnonClassField(public static final)
-                      ### DeclareAnnonClassField((private|protected) static final)
+                      ### DeclareAnonClassField(public static final)
+                      ### DeclareAnonClassField((private|protected) static final)
```

Comment:
This is apparently a currently incorrect thing:
https://github.com/sevntu-checkstyle/sevntu.checkstyle/blob/f74b8da56fbd63cbe5b0a2c5db229851c716c015/sevntu-checks/src/main/java/com/github/sevntu/checkstyle/checks/coding/CustomDeclarationOrderCheck.java#L206

I'll have to split it out (and send them a patch)

Reply:
Please revert this change now, it should be fixed in Check first